src/components/background/index.js

```diff
@@ -5,40 +5,27 @@ import injectSheet from 'react-jss';
 
 const styles = theme => ({
   wrapper: {
-    width: '100vw',
-    minHeight: '100vh',
-    flexDirection: 'column',
     backgroundImage: 'linear-gradient(to bottom, #114357, #f29492)',
-  },
-  boltzWrapper: {
-    marginTop: '25px',
-    marginBottom: '25px',
-    marginLeft: 'auto',
-    marginRight: 'auto',
-    position: 'relative',
-    alignItems: 'center',
+    flexDirection: 'column',
   },
   boltz: {
-    bottom: 0,
+    color: theme.colors.white,
     fontSize: '15px',
-    fontStyle: 'normal',
-    alignSelf: 'center',
-    lineHeight: 'normal',
     fontWeight: 'normal',
+    fontStyle: 'normal',
     fontStretch: 'normal',
+    lineHeight: 'normal',
     letterSpacing: 'normal',
-    verticalAlign: 'bottom',
-    color: theme.colors.white,
-    zIndex: 2,
+    alignSelf: 'center',
+    paddingBottom: '10px',
+    zIndex: 99999,
```

Comment:
`zIndex: 99999` means that the element will not be grayed out when opening the `WHY?` popup:

![image](https://user-images.githubusercontent.com/8138838/52994637-86fb8200-3418-11e9-9199-529a1422409a.png)

Also it causes the element to be above the FAQ box which looks a little weird.

![image](https://user-images.githubusercontent.com/8138838/52995272-69c7b300-341a-11e9-93fe-52e1819c8d13.png)

Reply:
Ill fix this in another PR for #95 I don't want to bloat this with too many unrelated changes.